.github/workflows/trivy-analysis.yml

```diff
@@ -0,0 +1,38 @@
+name: Trivy
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * MON"
+
+env:
+  REGISTRY: dataswift/hat
+
+jobs:
+  trivy-scan:
+    name: Scan
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Container - Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_DATASWIFT_USER }}
+          password: ${{ secrets.DOCKERHUB_DATASWIFT_PASS }}
+
+      - name: Container - Pull
+        run: docker pull ${{ env.REGISTRY }}:${{ github.sha }}
+
+      - name: Container - Scan
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: "${{ env.REGISTRY }}:${{ github.sha }}"
+          format: "template"
+          template: "@/contrib/sarif.tpl"
+          output: "trivy-results.sarif"
+          severity: "CRITICAL,HIGH"
```

Comment:
Just checking if we only want to know for critical and high. I would say all but if that's the requirement I am good

Reply:
@whiteshadow-gr up to you, just wanted to start with something and do not have false-positive results

Reply:
agree with @pperzyna, let's get it in and working, we can fine tune the config after.